semcore/card/src/Card.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import createComponent, { Component, sstyled, Root } from '@semcore/core';
 import { Box } from '@semcore/flex-box';
 import { Text } from '@semcore/typography';
-import Tooltip from '@semcore/tooltip';
+import { DescriptionTooltip } from '@semcore/tooltip';
```

Comment:
Looks like you forgot to update semcore/card/changelog.md file

Reply:
yes, sorry, added